src/Tests/Service/U2fServiceAuthenticationVerificationTest.php

```diff
@@ -252,4 +268,96 @@ class_exists('u2flib_server\U2F');
             [235789],
         ];
     }
+
+    /**
+     * @test
+     * @group authentication
+     * @dataProvider expectedDeviceErrors
+     *
+     * @param int $deviceErrorCode
+     */
+    public function it_handles_expected_device_errors($deviceErrorCode)
+    {
+        $keyHandle = 'key-handle';
+        $challenge = 'challenge';
+
+        $request = new \Surfnet\StepupU2fBundle\Dto\SignRequest();
```

Comment:
We've used aliasing before in other places, please do the same here. This may or may not be just as clear but deviates from what has been done before: use statements for everything (including core classes).